sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -134,10 +134,10 @@ search_metrics AS (
   FROM
     unioned
   LEFT JOIN
-    search_clients s
+    search_clients
   ON
-    unioned.client_id = s.client_id
-    AND unioned.submission_date = s.submission_date
+    unioned.client_id = search_clients.client_id
+    AND DATE_ADD(unioned.submission_date, INTERVAL 1 DAY) = search_clients.submission_date
```

Comment:
Seems like this is the major change at this point - could you link the discussion/bug about why this was necessary please? I remember seeing it but can't seem to find it haha

Reply:
We should definitely add some code comments here to link out to relevant discussions. It's going to be hard to maintain relevant context to understand why this adjustment exists.